### ISSUE TITLE
Reorder command sequence documented in contributing instructions

### ISSUE THREAD

**Issue by cyberladysj** [undp/SeleniumProbes#1]
- Reversing install procedure: Poetry should be installed first and then pre-commint would follow.
         "git clone https://github.com/undp/seleniumprobes.git SeleniumProbes
          cd SeleniumProbes
          **pre-commit install
          poetry install"**

- Include "pyenve" installation in ~/.bashrc, verifying virtual environment persistent installation.

**Comment by OK-UNDP** [comment]
@cyberladysj, thanks for spotting the doc error! 

Also, the third step of the [Managing Virtual Environments](../blob/master/CONTRIBUTING.md#tldr) section of [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md) mentions that `.bashrc` should be edited for `pyenv` to engage automatically. So, does this address your second bullet point or you meant something else?

**Comment by cyberladysj** [comment]
Yes, editing .bashrc for pyenve to engage automatically addresses my second bullet point. Thank you
